user-manuals/minigames/towercrush.md

```diff
@@ -0,0 +1,30 @@
+# Towercrush
+
+Towercrush is a minigame which is part of the Gamify-IT platform.
+
+## About the Game
+
+The goal of the game is to provide a simple game for students to practice their knowledge with single choice questions while competing against fellow students.
+
+![main menu screenshot](assets/towercrush-game.webp)
```

Comment:
Screenshot missing

Reply:
Will be added once we have improved the design of the game